src/main/webapp/WEB-INF/web.xml

```diff
@@ -76,4 +76,14 @@
     <url-pattern>/activity</url-pattern>
   </servlet-mapping>
 
+  <servlet>
+    <servlet-name>ProfileServlet</servlet-name>
+    <servlet-class>codeu.controller.ProfileServlet</servlet-class>
+  </servlet>
+
+  <servlet-mapping>
+    <servlet-name>ProfileServlet</servlet-name>
+    <url-pattern>Profile</url-pattern>
```

Comment:
Usually URLs are lower case, so it seems that `/profile` will be a more suitable pattern that `Profile`

- [ ] Update URL accordingly